composer.json

```diff
@@ -94,6 +94,7 @@
         "psr-4": {
             "Symfony\\Bridge\\Doctrine\\": "src/Symfony/Bridge/Doctrine/",
             "Symfony\\Bridge\\Monolog\\": "src/Symfony/Bridge/Monolog/",
+            "Symfony\\Bridge\\PhpUnit\\": "src/Symfony/Bridge/PhpUnit/",
```

Comment:
@nicolas-grekas 

Ok, I have two problems here:
- I can not test code with phpunit, because of error presented above
- I can not generate coverage reports using ./phpunit